src/intgtests/test_pam_hbac.py

```diff
@@ -333,6 +337,7 @@ def _driver_setup(self):
                                               admin_password,
                                               bind_dn,
                                               bind_pw,
+                                              start_ssl = True,
```

Comment:
On Mon, May 23, 2016 at 05:52:40AM -0700, Pavel Reichl wrote:
> > @@ -333,6 +337,7 @@ def _driver_setup(self):
> >                                                admin_password,
> >                                                bind_dn,
> >                                                bind_pw,
> > +                                              start_ssl = True,
> 
> PEP8 complains:
> src/intgtests/test_pam_hbac.py:340:56: E251 unexpected spaces around keyword / parameter equals
> src/intgtests/test_pam_hbac.py:340:58: E251 unexpected spaces around keyword / parameter equals
> 
> I think it would be nice to use same style as for 'insecure',

Yes, done.